uaa/src/test/java/org/cloudfoundry/identity/uaa/scim/endpoints/OpenIdConnectEndpointDocs.java

```diff
@@ -33,6 +33,7 @@ void getWellKnownOpenidConf() throws Exception {
                 fieldWithPath("claims_supported").description("JSON array containing a list of the Claim Names of the Claims that the OpenID Provider MAY be able to supply values for."),
                 fieldWithPath("claims_parameter_supported").description("Boolean value specifying whether the OP supports use of the claims parameter."),
                 fieldWithPath("service_documentation").description("URL of a page containing human-readable information that developers might want or need to know when using the OpenID Provider."),
+                fieldWithPath("code_challenge_methods_supported").description("<small><mark>UAA 4.31.0</mark></small>JSON array containing a list of [PKCE](https://tools.ietf.org/html/rfc7636) code challenge methods supported by this authorization endpoint."),
```

Comment:
next uaa will be 75.5.0, can you please update this , would cleanup the PR, thanks